package.json

```diff
@@ -189,7 +190,10 @@
     "lines": 84,
     "statements": 82,
     "functions": 83,
-    "branches": 69
+    "branches": 69,
+    "extension": [
+      ".js"
+    ]
```

Comment:
FYI: This is the `nyc` fix that I mentioned here:

https://github.com/knex/knex/pull/3613#issuecomment-579990257

Short story: it appears that `nyc`'s extension handling interferes with `Liftoff`'s extension handling.  Thus, when you run the unit tests under `nyc`, it fails to preload `ts-node` properly.